mesh/mesh.hpp

```diff
@@ -1944,6 +1954,17 @@ class Mesh
       operator Mesh::FaceInfo() const;
    };
 
+   /// Given a "face info int", return the face orientation. @sa FaceInfo.
+   static int DecodeFaceInfoOrientaiton(int info) { return info%64; }
```

Comment:
typo `orientaiton`

Reply:
Oops, thanks for pointing it out. Fixed in 76006e9b1.